src/commands/tag/README.md

```diff
@@ -29,10 +29,8 @@ datadog-ci tag --level job --tags "go.version:`go version`"
 
 ### Supported providers
 
-The tag command only works for the following CI providers: [Buildkite, CircleCI, GitHub, GitLab]. If used in
-any other provider it will fail. Note that for GitHub actions only the level `pipeline` is supported. If the
-command is invoked in GitHub actions with level `job` it will exit with status code 1 and return an
-error.
+The tag command only works for the following CI providers: [Buildkite, CircleCI, GitHub, GitLab, AzurePipelines, Jenkins]. If used in
```

Comment:
sorry, I didn't add the suggestion on this one 😟
```suggestion
The tag command only works for the following CI providers: Buildkite, CircleCI, GitHub, GitLab, Azure Pipelines and Jenkins. If used in
```

Reply:
And I didn't remember there were two very similar 😂 Committed!